lib/tabular_form_builder.rb

```diff
@@ -59,6 +59,18 @@ def label(method, text = nil, options = {}, &block)
     super
   end
 
+  def radio_button(field, value, options = {}, *args)
+    options[:class] = Array(options[:class]) + %w(form--radio-button)
+    label_options = options.dup
+    input_options = options.dup.except(:for, :label, :no_label)
+
+    label_options[:for] = "#{object_name}_#{field}_#{value.downcase}"
+    label = label_for_field(field, label_options)
+    input = super(field, value, input_options, *args)
+
+    (label + container_wrap_field(input, 'radio-button', options)).html_safe
```

Comment:
Copy'n'Paste issue. you are correct - we should probably also remove it from the generated methods of the form builder then, shouldn't we?